presidio-anonymizer/tests/anonymizers/test_validators.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from presidio_anonymizer.entities import InvalidParamException
+from presidio_anonymizer.anonymizers.validators import validate_parameter
+
+
+def test_validate_parameter_when_parameter_is_none_typed_ipe_raised():
+
+    with pytest.raises(InvalidParamException, match="Expected parameter name"):
+        validate_parameter(
+            parameter_value=None, parameter_name="name", parameter_type=int
+        )
+
+
+def test_validate_parameter_when_parameter_is_bad_typed_ipe_raised():
```

Comment:
Please use the convention agreed for unit tests e.g:
test_given<condition_to_test>_then<expected_results>

Reply:
Sure that we are dropping the method name? missed the then thanks